pkg/monitor/monitor/util.go

```diff
@@ -319,7 +331,118 @@ chmod 777 /data/prometheus /data/grafana
 	return container
 }
 
-func getMonitorPrometheusContainer(monitor *v1alpha1.TidbMonitor, tc *v1alpha1.TidbCluster) core.Container {
+func getMonitorDMInitContainer(monitor *v1alpha1.TidbMonitor, dc *v1alpha1.DMCluster) core.Container {
+	c := `mkdir -p /data/prometheus
+chmod 777 /data/prometheus
+/usr/bin/init.sh`
+	if monitor.Spec.Grafana != nil {
+		c = `mkdir -p /data/prometheus /data/grafana
+chmod 777 /data/prometheus /data/grafana
+/usr/bin/init.sh`
+	}
+	command := []string{
+		"/bin/sh",
+		"-c",
+		c,
+	}
+	alertManagerRulesVersion := dc.MasterImage()
+	if monitor.Spec.AlertManagerRulesVersion != nil {
+		alertManagerRulesVersion = fmt.Sprintf("dm:%s", *monitor.Spec.AlertManagerRulesVersion)
+	}
+	container := core.Container{
+		Name:  "monitor-dm-initializer",
+		Image: fmt.Sprintf("%s:%s", monitor.Spec.DMInitializer.BaseImage, monitor.Spec.DMInitializer.Version),
+		Env: []core.EnvVar{
+			{
+				Name:  "DM_CLUSTER_NAME",
+				Value: dc.Name,
+			},
+			{
+				Name:  "PROM_CONFIG_PATH",
+				Value: "/prometheus-rules",
+			},
+			{
+				Name:  "PROM_PERSISTENT_DIR",
+				Value: "/data",
+			},
+			{
+				Name:  "DM_VERSION",
+				Value: alertManagerRulesVersion,
+			},
+			{
+				Name:  "GF_DM_PROMETHEUS_URL",
+				Value: "http://127.0.0.1:9090",
+			},
+			{
+				Name:  "DM_CLUSTER_NAMESPACE",
+				Value: dc.Namespace,
+			},
+			{
+				Name:  "TZ",
+				Value: dc.Spec.Timezone,
```

Comment:
What if Timezone is not configured? Please define a function for this as the TidbCluster.

Reply:
use dc.Timezone() instead. When timezone is not configured, Timezonge() will return "UTC".